src/Symfony/Bridge/Doctrine/Middleware/ConnectionKeepAliveDriver.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\Bridge\Doctrine\Middleware;
+
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Driver\Connection as DriverConnection;
+use Doctrine\DBAL\Driver\Middleware\AbstractDriverMiddleware;
+
+class ConnectionKeepAliveDriver extends AbstractDriverMiddleware
+{
+    public function __construct(private Driver $driver, private \ArrayObject $connectionTimings, private string $connectionName = 'default')
```

Comment:
`$driver` must not be a promoted property, as we should not have a property declared in that class (the property is in the parent class)